src/transform3d.rs

```diff
@@ -51,16 +51,40 @@ impl<T, Src, Dst> TypedTransform3D<T, Src, Dst> {
     /// row-major convention) are the 13rd, 14th and 15th parameters.
     #[inline]
     pub fn row_major(
-            m11: T, m12: T, m13: T, m14: T,
-            m21: T, m22: T, m23: T, m24: T,
-            m31: T, m32: T, m33: T, m34: T,
-            m41: T, m42: T, m43: T, m44: T)
-         -> Self {
+        m11: T,
+        m12: T,
+        m13: T,
+        m14: T,
+        m21: T,
+        m22: T,
+        m23: T,
+        m24: T,
+        m31: T,
+        m32: T,
+        m33: T,
+        m34: T,
+        m41: T,
+        m42: T,
+        m43: T,
+        m44: T,
```

Comment:
Having the matrix members visually laid out helps me a lot (I can never remember m12 vs m21 etc.). That's the only thing that I would like to see formatted manually, but other than that the code looks a lot nicer, thanks!
Could you leave transform2d.rs and transform3d.rs out of this PR? I can go through these two files and selectively reformat them. Until we commit to automated reformatting or something that would really force us to lose the formatting of the parameters, I would prefer to preserve the visual layout of the members/arguments for now.